Classes/Frontend/PageTitleProvider/DateTitleProvider.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2023 Daniel Siepmann <coding@daniel-siepmann.de>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+namespace Wrm\Events\Frontend\PageTitleProvider;
+
+use Wrm\Events\Domain\Model\Date;
+
+final class DateTitleProvider implements DateTitleProviderInterface
+{
+    /**
+     * @var null|Date
+     */
+    private $date = null;
+
+    public function setDate(Date $date): void
+    {
+        $this->date = $date;
+    }
+
+    public function getTitle(): string
+    {
+        if ($this->date === null) {
+            return '';
+        }
+
+        return implode(' ', [
+            $this->date->getEvent()->getTitle(),
+            $this->date->getStart()->format('d.m.Y H:i'),
```

Comment:
This is a typical German format for the output of dates. Can this simply be made configurable, or does the extension use this date format so hard in various places?